win32ss/gdi/gdi32/objects/text.c

```diff
@@ -1,3 +1,10 @@
+/*
+ *  COPYRIGHT:        See COPYING in the top level directory
+ *  PROJECT:          ReactOS Kernel
+ *  PURPOSE:          GDI Support
+ *  FILE:             dll/win32/gdi32/objects/text.c
+ *  PROGRAMER:        Katayama Hirofumi MZ
```

Comment:
usually Programmer has two 'm' (in standard UK english).

Reply:
"Programer" seems to be acceptable spelling in the US. But it's not really relevant since our new file header uses COPYRIGHT instead (and that should be used, please):
https://github.com/reactos/reactos/blob/master/drivers/filesystems/fs_rec/cdfs.h#L1-L6

Reply:
OK.